arm_navigation_msgs/RobotTrajectory robot_trajectory
---
bool successful
